packages/itwinui-react/src/core/utils/components/Resizer.tsx

```diff
@@ -216,99 +234,80 @@ export const Resizer = (props: ResizerProps) => {
       <div
         data-iui-resizer='top-left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'nw-resize',
-        }}
+        style={{ cursor: 'nw-resize' }}
       />
       <div
         data-iui-resizer='top'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 'n-resize',
-        }}
+        style={{ cursor: 'n-resize' }}
       />
       <div
         data-iui-resizer='top-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'ne-resize',
-        }}
+        style={{ cursor: 'ne-resize' }}
       />
       <div
         data-iui-resizer='right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          right: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'e-resize',
-        }}
+        style={{ cursor: 'e-resize' }}
       />
       <div
         data-iui-resizer='bottom-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'se-resize',
-        }}
+        style={{ cursor: 'se-resize' }}
       />
       <div
         data-iui-resizer='bottom'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 's-resize',
-        }}
+        style={{ cursor: 's-resize' }}
       />
       <div
         data-iui-resizer='bottom-left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'sw-resize',
-        }}
+        style={{ cursor: 'sw-resize' }}
       />
       <div
         data-iui-resizer='left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          left: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'w-resize',
-        }}
+        style={{ cursor: 'w-resize' }}
       />
     </>
   );
 };
+
+// ----------------------------------------------------------------------------
+
+const ResizerStyles = React.memo(() => <style>{resizerStyles}</style>);
+
+const resizerStyles = /* css */ `
+[data-iui-resizer] {
+  pointer-events: auto;
+  grid-area: 1 / 1 / -1 / -1;
+  width: 12px;
+  height: 12px;
+  z-index: 1;
+}
+[data-iui-resizer='top'],
+[data-iui-resizer='bottom'] {
+  height: 8px;
```

Comment:
Since the edges and corners are now protruding out, would it be better to increase the size of the resizers so that they bleed into the dialog a little bit too? E.g. the resizer can be `x` pixels out and `x` pixels in the Dialog.

With this increased size of the resizers, would the resizing be easier for users?

Reply:
It's a tradeoff. We don't want it to protrude too much, because it could cover up other adjacent elements.

Reply:
non-blocking: I mean, earlier, the resizers were placed inside the dialog. But now they're only placed outside the dialog. So, I was thinking would it be better to also keep a reasonable amount inside the dialog. That reasonable amount could be how much was kept inside before this PR.

But how it is is okay too. Just sharing my thoughts. We can go with what you have currently too.

Reply:
Oh I misunderstood. Yes, we could make it go deeper inside the dialog, but I deliberately chose the current positioning, specifically because it would otherwise cover up the scrollbar (which is already super thin) and potentially the close button too.

<img width="55" alt="image" src="https://github.com/iTwin/iTwinUI/assets/9084735/be4a9d2a-b1c1-4105-a7e6-65f677246640">

Reply:
Ohh, I see, the current approach is safer too. So, keeping the current approach sounds good 👍.